README.md

```diff
@@ -96,6 +96,36 @@ UI.
 Conduit can be configured through command line parameters. To view the full list
 of available options, run `./conduit --help`.
 
+### Homebrew
+
+Make sure you have [homebrew](https://brew.sh/) installed on your machine, then run:
+
+```sh
+brew install conduit
+brew upgrade conduit
+```
+
+### Debian
+
+Download the right `.deb` file for your machine architecture from the
+[latest release](https://github.com/conduitio/conduit/releases/latest), then run:
+
+```sh
+apt-get install dpkg
+dpkg -i package_name.deb # replace with the downloaded package name
+apt-get update
+apt-get install conduit
+```
+
+### RPM
+
+Download the right `.rpm` file for your machine architecture from the
+[latest release](https://github.com/conduitio/conduit/releases/latest), then run:
+
+```sh
+sudo rpm -i package_name.rpm # replace with the downloaded package name
```

Comment:
Does it need `sudo`? Again, not a Linux expert 😄

Reply:
you need it only if you're not the root user, same for `dpkg .deb` file, but it should be common sense to use it if you don't have the permissions, so I'll delete it